store/tikv/client_batch.go

```diff
@@ -215,8 +215,8 @@ func (c *batchCommandsClient) send(request *tikvpb.BatchCommandsRequest, entries
 		c.batched.Store(requestID, entries[i])
 	}
 	if err := c.client.Send(request); err != nil {
-		logutil.BgLogger().Error(
-			"batch commands send error",
+		logutil.BgLogger().Info(
```

Comment:
which metric can we use to identity send failure if user use error/warn level

Reply:
Seems in the module there is no such metric. Is it better to put the metric into `SendRequest` callers?